solver/src/solver/http_solver.rs

```diff
@@ -368,47 +188,32 @@ impl HttpSolver {
             })
             .collect();
 
-        let mut new_price_estimates: HashMap<H160, f64> = queries
+        let mut all_price_estimates: HashMap<H160, f64> = queries
             .iter()
             .zip(new_price_estimates)
```

Comment:
Does new price estimates still exist? Seems weird it's rebranded and the old one is still being used.

Reply:
Previously, we had `price_estimates`, which were the price estimates for all the tokens included in GPv2 orders, `new_price_estimates` were price estimates for all the tokens in the batch (so, including tokens for from AMMs but that aren't included in any orders).

The previous code would combine `price_estimates` and `new_price_estimates` into `new_price_estimates`, which I thought was not an ideal name, and instead renamed it to `all_price_estimates` (since it is the combination of all the price estimates, the ones from the batch used for the objective function, and the extra ones that the HTTP solver wants).